Overview.src.html

```diff
@@ -2036,6 +2040,18 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <span title=concept-network-error>network error</span> and
   <var>response</var> is not a
   <span title=concept-filtered-response>filtered response</span>, set
+  <var>response</var>'s
+  <span title=concept-response-cors-exposed-headers-list>CORS-exposed headers list</span> to the result of
+  <span title=concept-header-parse>parsing</span>
+  `<code title=http-access-control-expose-headers>Access-Control-Expose-Headers</code>` in
+  <var>response</var>'s
+  <span title=concept-response-header-list>header list</span>.
+
+ <li>
+  <p>If <var>response</var> is not a
+  <span title=concept-network-error>network error</span> and
+  <var>response</var> is not a
+  <span title=concept-filtered-response>filtered response</span>, set
```

Comment:
I think we should try to find a solution that creates a CORS filtered response in a single step. Or at least a set of steps that belong together. E.g., the check for network error and filtered response is only needed once, you could then run substeps. But maybe there's a way to do the `<dl>` differently as well that cleans this up a bit.